### PR TITLE
TMDM-11566 Provide clear message when user input invalid system role name.

### PR DESCRIPTION
**Pull request by sbliu**
https://jira.talendforge.org/browse/TMDM-12056

**What is the current behavior?** (You should also link to an open issue here)
There is no info to show user why system role not be imported.
When user input invalid system role name、input 'system' name for role category, the error message is not accurate and clear.

**What is the new behavior?**
System role won't show on the import tree, and error messages will show in the  bottom 'Errors and Warnings' box of importing wizard page.

Give more clear message to user when input invalid system role name , input 'system' name when create role category.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/i18n/messages.properties [hunk 1]
```diff
@@ -473,4 +473,6 @@ _OkDeleteStatusMsg={0} "{1}" was deleted successfully
 _OkStatusMsg={0} "{1}" was deployed successfully
 _Resource=Resource
 _SubTaskTitle=Deleting {0}...
+cannotCreateSystemCategory="System"(case insensitive) category can not be created manually\!
```
Comment (pbabin) [inline main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/i18n/messages.properties]: Category name is invalid. 'System' is a reserved word.

File: main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/i18n/messages.properties [hunk 2]
```diff
@@ -473,4 +473,6 @@ _OkDeleteStatusMsg={0} "{1}" was deleted successfully
 _OkStatusMsg={0} "{1}" was deployed successfully
 _Resource=Resource
 _SubTaskTitle=Deleting {0}...
+cannotCreateSystemCategory="System"(case insensitive) category can not be created manually\!
 NO_XSD_RESOURCE=no datamodel resouce
+shouldNotBeSystemRoleName=The name must not start with "system_" and not be "administration" (case insensitive)\!
```
Comment (pbabin) [inline main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/i18n/messages.properties]: Role name is invalid. Name cannot start with 'System_' nor be 'administration'

Comment (pbabin) [reply main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/i18n/messages.properties]: It's not specific to this case, all custom role names are case-insensitive.

Comment (sbliu) [reply main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/i18n/messages.properties]: ok

File: main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/utils/RepositoryResourceUtil.java [hunk 3]
```diff
@@ -415,6 +415,15 @@ public static IFolder getFolder(ERepositoryObjectType type) {
         return objectFolder;
     }
 
+    public static boolean isSystemViewObject(Item item) {
```
Comment (hbhong) [inline main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/utils/RepositoryResourceUtil.java]: the method is named "isSystemViewObject" but input parameter is "Item" type, rename to "isSystemView" maybe better.

File: main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/utils/RepositoryResourceUtil.java [hunk 4]
```diff
@@ -415,6 +415,15 @@ public static IFolder getFolder(ERepositoryObjectType type) {
         return objectFolder;
     }
 
+    public static boolean isSystemViewObject(Item item) {
+        String path = item.getState().getPath().toLowerCase();
```
Comment (hbhong) [inline main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/utils/RepositoryResourceUtil.java]: when "item.getState().getPath()" return null, this statement will throws NPE

**Pull request merged**